README.md

```diff
@@ -1,2 +1,23 @@
-# shop-migration
-Migration scripts from legacy database for Magento2 used for RWAHS shop
+# RWAHS Shop Migration
+
+Migration scripts from legacy database for Magento2 used for RWAHS shop.
+
+The overall flow of data for the migration process is as follows:
+
+1. Use `mdbtools` commands to extract data from legacy Access database into CSV files.
+2. Use custom `node` scripts to convert the legacy CSV data into a CSV format compatible with Magento2.
+3. Import the processed CSV into Magento2 using its CSV import functionality.
+
+## Prerequisites
+
+* `mdbtools` can be installed via `apt-get`.
+* `node` is recommended to be installed either via `nvm` (https://github.com/creationix/nvm) or the official PPAs
```

Comment:
`APT repositories` not `PPAs` as a PPA is specifically something hosted on launchpad.net.